src/server/game/Cache/CharacterCache.cpp

```diff
@@ -0,0 +1,391 @@
+/*
+ * This file is part of the AzerothCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by the
+ * Free Software Foundation; either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "CharacterCache.h"
+#include "ArenaTeam.h"
+#include "DatabaseEnv.h"
+#include "Log.h"
+#include "Player.h"
+#include "Timer.h"
+#include "World.h"
+#include "WorldPacket.h"
+#include <unordered_map>
+
+namespace
+{
+    std::unordered_map<ObjectGuid, CharacterCacheEntry> _characterCacheStore;
+    std::unordered_map<std::string, CharacterCacheEntry*> _characterCacheByNameStore;
+}
+
+CharacterCache::CharacterCache()
+{
+}
+
+CharacterCache::~CharacterCache()
+{
+}
```

Comment:
delete this and in `.h` file can
```cpp
CharacterCache() noexcept = default;
~CharacterCache() noexcept = default;
```

Reply:
Thanks, I had gotten that error and had no time to look into it, it should be fine now